op-e2e/actions/interop/interop_test.go

```diff
@@ -572,27 +571,25 @@ func TestInteropFaultProofs(gt *testing.T) {
 			// The derivation reaches the L1 head before the next block can be created
 			l1Head:         actors.L1Miner.L1Chain().Genesis().Hash(),
 			expectValid:    true,
-			skipChallenger: true, // Challenger doesn't yet check if blocks were safe
+			skipChallenger: true, // test's agreedClaim is incorrect - first chain is also invalid
```

Comment:
Is the `skipChallenger: true` here intentional? What functionality is missing for the challenger to handle this case?

Reply:
Yeah this one still doesn't work because the challenger assertions check it creates the same agreed state.  But the test is actually using an incorrect prestate - both chains are missing L1 data so it the state transition actually goes to the invalid hash at the first chain's optimistic block and this test is claiming it was still valid then. That works for op-program but not challenger.

We need to be able to submit the batch data to different L1 blocks to be able to test this case in challenger.